### DISCUSSION TITLE
Pixel 2 XL battery idle details

### DISCUSSION THREAD

**Discussion by cad0p** [VR-25/acc#131] [Q&A] [answered]
Hi, I'm considering to root my device only for this module (thanks for your work!)

I saw that in the code these lines below feature a specific implementation of idle mode for the Pixel 2 XL

https://github.com/VR-25/acc/blob/dev/acc/oem-custom.sh#L24-L26

Is this idle mode aware of the battery percentage (voltage)? What is the preferred configuration for the Pixel 2 XL?

Thanks a lot!

**Comment by VR-25** [comment, marked as answer]
Hi,
According to a user who tested it, this idle mode implementation works just like regular idle mode from other devices.
To set things up, you just have to run `acc -t` to see the switch [group] that supports idle mode, then run `acc -ss` to enforce it.
